free-programming-books.md

```diff
@@ -1524,7 +1524,7 @@ See also [TeX](#tex)
 ### Python
 * [A Beginner's Python Tutorial](http://en.wikibooks.org/wiki/A_Beginner%27s_Python_Tutorial)
 * [A Bit of Python and Other Things](https://github.com/RafeKettler/magicmethods)
-* [A Guide to Python's Magic Methods](http://www.rafekettler.com/magicmethods.html) - Rafe Kettler
+* [A Guide to Python's Magic Methods](https://github.com/RafeKettler/magicmethods) - Rafe Kettler
```

Comment:
Great catch, thanks!

[Parent commit](https://github.com/vhf/free-programming-books/commit/1639af6ed32c9d993bc98ba9b8216ca71d915639#diff-09e7ccff8516d5272cc11dcbff51c4bbR1526) misplaced this link, which is why it's now a duplicated URL with different names.

I don't know how to reintroduce `[A Bit of Python and Other Things](http://jessenoller.com/good-to-great-python-reads/)` though, because it's not really a book. I'll think about it.